vscode-dotnet-runtime-library/src/Utils/CommandExecutor.ts

```diff
@@ -347,6 +365,7 @@ ${stderr}`));
         await executeWithLock(this.context.eventStream, false, RUN_UNDER_SUDO_LOCK(this.sudoProcessCommunicationDir), SUDO_LOCK_PING_DURATION_MS, this.context.timeoutSeconds * 1000 / 5,
             async () =>
             {
+                await (this.fileUtil as FileUtilities).wipeDirectory(this.sudoProcessCommunicationDir, this.context?.eventStream, ['.txt']);
```

Comment:
What's the `.txt` referring to here? I'm a bit confused about what is exactly being written to disk.

Reply:
The `.txt` files are there to communicate with the sudo process. We can't use fork to talk to it, because the sudo library does not provide a method to do so. Probably it would have been easier at this point to go edit the vscode sudo library, but alas